### ISSUE TITLE
Changes to AMO detail pages when visited from non-Firefox browser

### ISSUE THREAD

**Issue by atsay** [mozilla/addons-frontend#5245]
### Describe the problem and steps to reproduce it:
We need to change a link and add a link to AMO detail pages that are reached via non-Firefox browsers. 

### What happened?
When you reach an AMO detail page from a non-Firefox browser, you will see the green "Only with Firefox--Get Firefox Now" button.

Currently, it links to https://www.mozilla.org/firefox/new/?utm_source=addons.mozilla.org&utm_medium=referral&utm_campaign=non-fx-button&utm_content=install-addon-button

However, this link requires the user to click again to install Firefox.

### What did you expect to happen?

We need to make the following changes:

1) Link to https://www.mozilla.org/firefox/download/thanks/?utm_source=addons.mozilla.org&utm_medium=referral&utm_campaign=non-fx-button&utm_content=install-addon-button (per comment below), which will start the download automatically.

2) Add a link to the **Firefox Privacy Notice** under the green "Only with Firefox--Get Firefox Now" button: https://www.mozilla.org/privacy/firefox/

### Anything else we should know?
@pwalm Can you please provide guidance on color, positioning, and text size of the Privacy Policy? Perhaps similar to this? ![privacy policy link](https://i.imgur.com/80VzOow.png) You can see this live by going to https://www.mozilla.org/firefox/facebookcontainer/ from a non-Firefox browser.

**Comment by chrismore** [comment]
@atsay Actually the ?scene=2 has been deprecated since the site has changed over to /download/thanks/ as scene 2 so the URL should be:

https://www.mozilla.org/firefox/download/thanks/?utm_source=addons.mozilla.org&utm_medium=referral&utm_campaign=non-fx-button&utm_content=install-addon-button

**Comment by atsay** [comment]
Thanks, @chrismore!

**Comment by chrismore** [comment]
We should get @hoosteeno to double check this too.

**Comment by hoosteeno** [comment]
TLDR, r+.

I mentioned this over in #websites. 

There are a few concerns about sending people directly to download from a non-www site, because being able to download Firefox from non-www sites is a malware vector that we've discouraged in part by trying to be rigorous about where you can get the official download. The UX benefit of this change might override those concerns, but we generally prefer not to send people straight to download from anywhere but www.

agibson rightly pointed out that various legacy browsers don't get the auto-download experience when they land on /thanks (they only get it if they the our highly engineered download button on www). Instead, legacy browser users going through this experience will land on a page saying thanks, and will have to click again to trigger the download. It looks like we've had roughly 5k visits from IE versions <= 9 in 2018, which is a pretty tiny sliver of AMO's audience. Again, the UX benefit here might override this concern.

We don't expect any negative SEO ramifications, but we'll watch. cc'ing @rraue so he can see when this lands.

**Comment by atsay** [comment]
Great, we'll go forward with getting some UX guidance first.

**Comment by pwalm** [comment]
I thought I gave my guidance somewhere along the way on this, but it should just replace the "Add to Firefox" button on an add-on detail page, and should follow the [primary button spec states](https://design.firefox.com/photon/components/buttons.html#primary-3) (but [change the colour to green](https://design.firefox.com/photon/visuals/color.html#green)).

@MeridelW Can you provide guidance on the button copy? I know you had some issues with it awhile back.

**Comment by pwalm** [comment]
Oh, @atsay pointed out that I missed the "privacy" infor below the button. Follow the same guidance as the image further up this thread, but set it in our standard link blue.

**Comment by jvillalobos** [comment]
If there's new copy available at time of implementation, we'll use the new copy. Otherwise I'd rather not block on it. Bumping up to P2 since this is a significant source of Firefox downloads.

**Comment by pwalm** [comment]
I think one of the issues was that the original string had an exclamation mark, but that goes against our copy guidelines. So at the very least, strip that out: "Only with Firefox - Get Firefox Now"

**Comment by atsay** [comment]
@pwalm you might have misunderstood the issue--we're not making UX changes to that green button. We're simply changing the button's link, and adding a "Firefox Privacy Notice" link below it. The UX guidance we need from you is what the color, size, and positioning should be for the "Firefox Privacy Notice" link. @MeridelW probably doesn't need to be pulled in.

**Comment by pwalm** [comment]
Oh, my mistake. Follow the positioning above, and use the [Caption 10 Bold styling on the Web typography](https://design.firefox.com/photon/visuals/typography.html#scale) scale tab.

**Comment by ddurst** [comment]
Let's hold off on this for now, as it's been confirmed that changing the link target will break stub attribution (as it is now). If we want to change it, we need to coordinate that with the stub attribution process and owners.

**Comment by atsay** [comment]
Blocked on https://bugzilla.mozilla.org/show_bug.cgi?id=1468680

**Comment by kumar303** [comment]
Why is getting desktop attribution (source of extension that prompted install) a blocker? That sounds like a nice enhancement but users would still benefit from an immediate download without it, no?

**Comment by jvillalobos** [comment]
Getting attribution info is required for the Return to AMO project to work. So, if we change this now to a direct install, we'll probably need to change it _back_ later in the quarter.

**Comment by muffinresearch** [comment]
@ddurst, @jvillalobos Do we still need to block block on landing what's been proposed in https://github.com/mozilla/addons-frontend/pull/5401 ? 

I think the attribution changes are filed in a separate bug here: https://github.com/mozilla/addons-frontend/issues/5374

**Comment by jvillalobos** [comment]
Since the plan for RtA is still to use attribution, and these changes will break it, I think it's safe to close this issue for now. If we end up doing something different that isn't broken by this change, we can reopen.

**Comment by willdurand** [comment]
> Since the plan for RtA is still to use attribution, and these changes will break it, I think it's safe to close this issue for now. If we end up doing something different that isn't broken by this change, we can reopen.

Should we close the PR as well? /cc @kumar303

**Comment by jvillalobos** [comment]
I think so, yes.

**Comment by kumar303** [comment]
Done. For future reference, the patch was here https://github.com/mozilla/addons-frontend/pull/5401

**Comment by willdurand** [comment]
@muffinresearch do we want to reopen this issue?

**Comment by muffinresearch** [comment]
@willdurand @ddurst can correct me if I'm wrong, but I think the only change required for this project  was the attribution link.
